File: src/main/kotlin/com/huawei/cangjie/idea/icons/CangJieResourcesIcons.java
package com.huawei.cangjie.idea.icons;

import com.intellij.ui.IconManager;
import org.jetbrains.annotations.NotNull;

import javax.swing.*;

public final class CangJieResourcesIcons {
    private static @NotNull Icon load(@NotNull String path, int cacheKey, int flags) {
        return IconManager.getInstance().loadRasterizedIcon(path, CangJieResourcesIcons.class.getClassLoader(), cacheKey, flags);
    }
    /** 16x16 */ public static final @NotNull Icon CangJie_file = load("/icons/cangjie_file.svg", 486618922, 0);

}

File: src/main/kotlin/com/huawei/cangjie/icon/CangJieResourcesIcons.kt
package com.huawei.cangjie.icon

import com.intellij.ui.IconManager

import javax.swing.Icon

object CangJieResourcesIcons {
    private fun load(path: String, cacheKey: Int, flags: Int): Icon {
        return IconManager.  getInstance().loadRasterizedIcon(
            path,
            CangJieResourcesIcons::class.java.classLoader, cacheKey, flags
        )
    }

    /** 16x16  */
    val CangJie_file: Icon = load("/icons/cangjie_file.svg", 486618922, 0)
}
